### ISSUE TITLE
Satellite paths

### ISSUE THREAD

**Issue by rickid123** [vasturiano/react-globe.gl#141]
How can I implement satellite paths/orbits into the path layer? I have based my project off of the satellites example.

**Comment by vasturiano** [comment]
Well, you just need to pass your orbits coordinates into the paths layer, via `pathsData`.

**Comment by rickid123** [comment]
That is about as far as I got with working out how to do it but i'm not too sure how to prepare the data for it. I am using satellite.js. I have been trying to base it around the solution you provided here [ https://github.com/vasturiano/react-globe.gl/issues/36#issuecomment-804378183 ] but haven't managed to get it working for paths around the globe. . I suspect this is more of an understanding of satellite.js where i'm lacking but any pointers would be helpful.

**Comment by vasturiano** [comment]
Yes, I'd recommend getting familiar with [satellite-js](https://github.com/shashwatak/satellite-js) to better understand how to retrieve satellite path coordinates from TLE files.
